lambda/resize.js

```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const childProcess = require('child_process');
 const fs = require('fs');
 const util = require('util');
+const gifsicle = require('gifsicle');
```

Comment:
i dont see this being used? (im slightly blind)

Reply:
its a WIP.